docs/source/blocks/block-style-wrapper.md

```diff
@@ -165,6 +167,54 @@ data like:
 
 will generate classnames `primary inverted`
 
+## Injecting custom CSS properties
+
+The style wrapper also allows to inject custom CSS properties, using the converter syntax.
+This use case is useful in some scenarios where the property that you are injecting is generic, customizable per project.
```

Comment:
I'm not sure whether "generic" and "customizable" should be used in the same sentence. Either the class is generic or customized, but not both at the same time. Can you clarify which it is or is it either one or the other?

```suggestion
This use case is useful in the scenario where the property that you want to inject is either generic or is customizable per project.
```

Reply:
I did frown too but it's what it is... something ready to be customized, then it's generic, right?

Reply:
Something ready to be customized is "customizable". Generic means general or lacking specificity. I edited my suggestion accordingly and made it more concise.